contributors/devel/api_changes.md

```diff
@@ -479,7 +508,30 @@ use our own equivalents for JSON serialization. The `pkg/api/serialization_test.
 will verify that your protobuf serialization preserves all fields - be sure to
 run it several times to ensure there are no incompletely calculated fields.
 
-## Edit json (un)marshaling code
+### Generate Clientset
+
+See [this document](generating-clientset.md) specific to generating client sets.
+
+### Generate Listers
+
+`lister-gen` is a tool to generate listers for a client. It requires a
+client generated by `client-gen`.
+
+`lister-gen` requires the `// +genclient=true` annotation on each
```

Comment:
Global resources also need `// +nonNamespaced=true` as a directive.

Also, these annotations are required for client-gen as well, and the linked document doesn't describe them.  I'm starting to wonder whether the generating-clientset.md file should just be generating-code.md and have doc for all the things there.